docs/community/governance.md

```diff
@@ -0,0 +1,180 @@
+
+## Decision making
+
+### Maintainers Team
+
+Team member status may be given to those who have made ongoing contributions to the Observatorium project for at least 3 months.
+This is usually in the form of code improvements and/or notable work on documentation, but organizing events or user support could also be taken into account.
+
+New members may be proposed by any existing Maintainer by email to [Observatorium][team]. It is highly desirable to reach consensus about acceptance of a new member.
+However, the proposal is ultimately voted on by a formal [supermajority vote](#supermajority-vote) of Team Maintainers.
+
+If the new member proposal is accepted, the proposed team member should be contacted privately via email to confirm or deny their acceptance of team membership.
+This email will also be CC'd to [Observatorium][team] for record-keeping purposes.
+
+If they choose to accept, the following steps are taken:
+
+* Maintainer is added to the [GitHub organization][gh] as _Owner_.
+* Maintainer is added to the [Observatoriumo][team].
+* Maintainer is added to the list of team members [here][maintainers-doc]
+* New maintainer is announced on the [Observatorium Twitter](https://twitter.com/0bservatorium) by an existing team member.
+
+Team members may retire at any time by emailing [observatorium-team@googlegroups.com][team].
+
+Team members can be removed by [supermajority vote](#supermajority-vote) on [observatorium-team@googlegroups.com][team]. For this vote, the member in question is not eligible to vote and does not count towards the quorum.
+
+Upon death of a member, their team membership ends automatically.
+
+### Technical decisions
+
+Smaller technical decisions are made informally and [lazy consensus](#consensus) is assumed. Technical decisions that span multiple parts of the Observatorium project
+should be discussed and made on the [GitHub issues][issues] and in most cases followed by proposal as described [here](/CONTRIBUTING.md#adding-new-features--components).
+
+Decisions are usually made by [lazy consensus](#consensus). If no consensus can be reached, the matter may be resolved by [majority vote](#majority-vote).
+
+### Governance changes
+
+Material changes to this document are discussed publicly on the [Observatorium GitHub](http://github.com/observatorium/observatorium).
+Any change requires a [supermajority](#supermajority-vote) in favor. Editorial changes may be made by [lazy consensus](#consensus) unless challenged.
+
+### Other matters
+
+Any matter that needs a decision, including but not limited to financial matters, may be called to a vote by any Maintainer if they deem it necessary.
+For financial, private, or personnel matters, discussion and voting takes place on the [observatorium-team@googlegroups.com][team]; Otherwise discussion and votes are held in public on the GitHub issues or #observatorium-dev CNCF slack channel.
```

Comment:
```suggestion
For financial, private, or personnel matters, discussion and voting takes place on the [observatorium-team@googlegroups.com][team]; mailing list Otherwise discussion and votes are held in public on the GitHub issues or #observatorium-dev CNCF slack channel.
```